netsocadmin/templates/page-skeleton.html

```diff
@@ -268,14 +270,20 @@
                                 <li><a href="#!">Change Password</a></li> -->
                             </ul>
                         </li>
-                        {% endif %}
                         <li {% if active == "home" %} class="active" {% endif %} title="The homepage of Netsoc Admin" style="margin-top: 2em">
                             <a style="width: 100%" href="/tools">
                                 <i class="fas fa-home">
                                     <span>Home</span>
                                 </i>
                             </a>
                         </li>
+                        <li {% if active == "account" %} class="active" {% endif %} title="Manage your netsoc server account">
```

Comment:
Very tiny, just a capital N for netsoc. Also at the other place where this string is